views/team.html

```diff
@@ -58,21 +56,15 @@
       <p class="vignette"></p>
     </div>
   </section>
-  <section class="block-section">
-    <div class="partners-container">
-      <p class="vignette"></p>
-    </div>
-    <div class="space"></div>
-    <div class="text-container">
-      <h3>PARTNERS</h3>
-      <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.</p>
-    </div>
 
-  </section>
+  <div class="partners">
+    <h3>PARTNERS</h3>
+  </div>
   
     <div class="footer">
       <div class="logo">
         <span id="navy"> BRUIN</span><span> ENTREPRENEURS </span>
+        <div class="love">Made with <i class="fa fa-heart-o" aria-hidden="true"></i> in Los Angeles </div>
```

Comment:
what the f is    fa fa-heart-o

Reply:
Thats font awesome, which is where the heart icon comes from when you open it up